--- .bashrc	2013-08-22 15:43:28.357704000 -0400
+++ .bashrc.uw	2013-08-22 15:53:47.465884000 -0400
@@ -85,20 +85,30 @@
     source ~/.bash_aliases
 fi
 
 # enable programmable completion features (you don't need to enable
 # this, if it's already enabled in /etc/bash.bashrc and /etc/profile
 # sources /etc/bash.bashrc).
 if [ -f /etc/bash_completion ] && ! shopt -oq posix; then
     . /etc/bash_completion
 fi
 
+if [ -d /u/cs350 ]; then
+	CLASSPATH=:/u/cs350/sys161/bin:/u/cs350/bin
+	export PATH=$PATH:$HOME/bin$CLASSPATH
+fi
+
+if [ -e /u/cs241/setup ]; then
+	source /u/cs241/setup
+fi
+
+
 export INPUTRC=$HOME/.inputrc
 export EDITOR=vim
 
 # Set shared library path
 export LD_RUN_PATH=$HOME/lib:/usr/local/lib:/usr/lib
 
 if [ -n "$FORTUNE" ]; then
 	wait $FORTUNE
 	echo "Hello Eric. Welcome to $(hostname)."
 fi
